packages/rum_sdk/lib/rum_flutter.dart

```diff
@@ -160,6 +160,14 @@ class RumFlutter {
     _instance.meta =
         Meta.fromJson({..._instance.meta.toJson(), 'user': userMeta.toJson()});
     _instance._batchTransport?.updatePayloadMeta(_instance.meta);
+
+    if (userId != null || userName != null || userEmail != null) {
+      RumFlutter().pushEvent('user_updated', attributes: {
```

Comment:
2 thoughts:

1: maybe we should prefix our internal Faro-events with something `faro_internal_{the event name}`? 
to avoid that a user of the SDK accidentally creates the same event..?

2: Should we send the 
`RumFlutter().pushEvent('user_updated' ..)`-event even if the userId is empty maybe?
I mean, it could be that the user logs out? In which case we want to record that the user is now "empty"

what do you think?